### PR TITLE
remove resource owners from scopes

### PR DESCRIPTION
**Pull request by ryanfkeepers**
## Description
    
Now that resource owners are identified via
the selector itself, rather than each scope, we
can remove the resource owner data from
scope production and data.

## Does this PR need a docs update or release note?

- [x] :no_entry: No 

## Type of change

- [x] :sunflower: Feature

## Issue(s)

* #1617

## Test Plan

- [x] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

File: src/cli/utils/exchange.go [hunk 1]
```diff
@@ -75,11 +75,11 @@ func AddExchangeInclude(
 	containsFolders, prefixFolders := splitFoldersIntoContainsAndPrefix(folders)
 
 	if len(containsFolders) > 0 {
-		sel.Include(eisc(resource, containsFolders, items))
+		sel.Include(eisc(containsFolders, items))
```
Comment (dadams39) [inline src/cli/utils/exchange.go:78]: L67 & 68 probably need to be removed as resource is no longer used.

**Comment by dadams39** [comment]
Rebase required.

**Review by ashmrtn — APPROVED** [review]

File: src/internal/connector/graph_connector_disconnected_test.go [hunk 2]
```diff
@@ -216,8 +216,9 @@ func (suite *DisconnectedGraphConnectorSuite) TestVerifyBackupInputs() {
 			name:       "Valid Single User",
 			checkError: assert.NoError,
 			getSelector: func(t *testing.T) selectors.Selector {
-				sel := selectors.NewExchangeBackup([]string{"bobkelso@someHospital.org"})
-				sel.Include(sel.MailFolders([]string{"bobkelso@someHospital.org"}, selectors.Any()))
+				sel := selectors.NewExchangeBackup([]string{"bobKelso@someHospital.org"})
+				sel.DiscreteOwner = "bobKelso@someHospital.org"
```
[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/connector/graph_connector_disconnected_test.go:220]: Yes and no.  If there's only one owner specified in the constructor, it auto-populates the `DiscreteOwner` value.  So in this case it's not needed (thought I deleted it here, might be a rebase mistake).

File: src/internal/connector/graph_connector_disconnected_test.go [hunk 3]
```diff
@@ -226,11 +227,21 @@ func (suite *DisconnectedGraphConnectorSuite) TestVerifyBackupInputs() {
...
 				sel.DiscreteOwner = "janitor@someHospital.org"
 				return sel.Selector
 			},
 		},
+		{
+			name:       "Invalid discrete owner",
+			checkError: assert.Error,
+			getSelector: func(t *testing.T) selectors.Selector {
+				sel := selectors.NewOneDriveBackup([]string{"elliotReid@someHospital.org", "janitor@someHospital.org"})
+				sel.DiscreteOwner = "janitor@someHospital.org"
```
[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/connector/graph_connector_disconnected_test.go:240]: But in this case there are two owners in the constructor, so for this test case we manually specify the discrete owner.  An sdk user would be expected to call `SplitByResourceOwner()` and iterate through the results, each of which will have that value populated separately.

[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/connector/graph_connector_disconnected_test.go]: I updated the comment on the DiscreteOwner selector property to better explain expectations here.

There's room for future security to unexport the value, provide a getter, and expect tests to call SplitByResourceOwners(), and take the selector with the discreteOwner they want to test on.  But for the moment I'm not going to prioritize the change.

**Pull request merged**